ganttproject/src/net/sourceforge/ganttproject/gui/taskproperties/TaskAllocationsPanel.java

```diff
@@ -122,6 +125,21 @@ public void changeValue(ChangeValueEvent event) {
     return result;
   }
 
+  private JComponent createLoadPanel() {
+    OptionsPageBuilder builder = new OptionsPageBuilder();
+
+    JPanel optionsPanel = new JPanel();
```

Comment:
This code is either over- or under- engineered :)
`GPOption` and `GPOptionGroup` classes serve as UI-independent "properties", or as a model for UI components if you wish. `OptionsPageBuilder` can automatically build UI from options and groups, adds titles, makes them readonly depending on the model properties. If you use this machinery then it makes sense to utilize it completely and do not bother with UI components explicitly (and it is the recommended way, unless UI is more complex than property sheet -like)

In order to use it, remove all code at lines 130-140 and replace it with 
```
return builder.createGroupComponent(myLoadGroup);
```
Also, add the option instance:
```
  private final DefaultDoubleOption myLoadOption = new DefaultDoubleOption("taskProperties.load.value");
```
and pass it to `myLoadGroup` in the group constructor call.

You're almost done, the remaining thing is to set the value of `myLoadOption` from `myTask`